spec.md

```diff
@@ -115,6 +116,19 @@ Return a list of counties in `Division` data structure.
 A `prefecture_code` is a 6-length code that endswith `00`. It can also be a 4-length code.
 
 
+### `#revisions`
+
+List all revisions supported by GB2260. This is a module(class) constant.
+
+
+### `#search(code, str[] revisions)`
```

Comment:
By this spec, `GB2260.search('123456', None)` is equivalent to `GB2260.search('123456', GB2260.revisions)`. I think that's indeed single-responsibility. If the implementation supports default argument or function overloading, then it would be better written as `GB2260.search('123456')`.

Returning a list of matched items is interesting. Maybe it's a better choice because a divison may be revoked by one revision and later be establish again by another, in theory. A list of matched items can provide this kind of info.